swabbie-redis/src/main/kotlin/com/netflix/spinnaker/swabbie/redis/RedisResourceStateRepository.kt

```diff
@@ -112,6 +119,17 @@ class RedisResourceStateRepository(
     }
   }
 
+  override fun remove(resourceId: String, namespace: String) {
+    "$namespace:$resourceId".let { id ->
+      statesKey(id).let { key ->
```

Comment:
Doesn't look like `key` is used anywhere here? Are we supposed to be using `key` in the redis commands instead?

Reply:
good call, we are not. removing.